ihmc-perception/src/main/java/us/ihmc/perception/detections/yolo/YOLOv8DetectionExecutor.java

```diff
@@ -137,28 +137,29 @@ public void runYOLODetectionOnAllModels(RawImage colorImage, RawImage depthImage
 
    /**
     * Non-blocking call to run YOLO on the provided images
-    * @param colorImage BGR color image, used for YOLO detection
+    * @param colorImage RGB color image, used for YOLO detection
     * @param depthImage 16UC1 depth image, used to get points of detected objects
     */
    public void runYOLODetection(YOLOv8ObjectDetector yoloDetector, RawImage colorImage, RawImage depthImage)
    {
       if (yoloDetector.isReady() && !yoloExecutorService.isShutdown())
       {
+         // Acquire the images
+         if (colorImage.get() == null || depthImage.get() == null)
```

Comment:
Moved calls to `get` here so it guarantees you "get" the images before they're deallocated.